app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -344,4 +356,42 @@ def stale_cache?(last_refresh_time)
       last_refresh_time > current_val[:created_at]
     end
   end
+
+  private
+
+  #
+  # Checks if IP address to host name resolving is enabled.
+  #
+  # @return [Boolean] `true` if host name resolving is enabled in the configuration, `false` otherwise.
+  #
+  def resolve_ip_addresses?
+    ::Settings.ems.ems_redhat.resolve_ip_addresses
+  end
+
+  #
+  # Tries to convert the given IP address into a host name, doing a reverse DNS lookup if needed. If it
+  # isn't possible to find the host name the original IP address will be returned, and a warning will be
+  # written to the log.
+  #
+  # @param address [String] The IP address.
+  # @return [String] The host name.
+  #
+  def resolve_ip_address(address)
+    require 'resolv'
```

Comment:
I dont think you need this because you are using stdlib stuff - but then I see it other places too.
@chrisarcand is there a reason to require here? like we do `require 'ostruct' [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/host.rb#L1)